modules/js.js

```diff
@@ -22,6 +22,69 @@ const js = {
             await callback(array[index], index, array);
         }
     },
+
+    /**
+     * Blocks for a given duration
+     * @param {number} milliseconds number of milli-seconds to wait
```

Comment:
I _believe_ we have a `sleep` function with a very similar/identical implementation in `CoreUtils` from andculturecode-javascript-core - but double check

Reply:
doh 🤦 - blame it on being on vacation